reedstrm/baking_workflow_and_errors.md

```diff
@@ -0,0 +1,120 @@
+# Baking and Recipes
+
+Baking has become a critical step for publishing - however, it is not planned for it to cause publishing to fail. This is partly due to how long baking may take - much like PDF generation, it can take several hours, if there are external parts, like exercises, that need to be fetched and incorporated into the pages.
```

Comment:
On further thought, pages will appear immediately on publish, since there is no such thing as a baked page outside of a book context. Books will not move to latest until they succeed in baking.